pyproject.toml

```diff
@@ -52,9 +52,6 @@ Source = "https://github.com/ipython/ipython"
 Tracker = "https://github.com/ipython/ipython/issues"
 
 [project.optional-dependencies]
-black = [
```

Comment:
hm, as long as this is still in there... https://github.com/ipython/ipython/blob/aedea319404107bfec9f7c70356cbba68790c06e/IPython/terminal/interactiveshell.py#L136

Reply:
I was not sure about this one, it's not enabled by default because broken, and should we really have a `ipython[black]`, if we can just pip install black ?

Reply:
Welp, if it's broken, and not tested, a major bump is a great time to take it out!

`[extras]` _are_ a good way to have a declarative compatibility range, but as `pip` mostly disregards them on subsequent solves, they aren't good for much in onesy-twosey interactive package management. :shrug:

Reply:
you are right i'll probably will remove the black hook